js/card.js

```diff
@@ -10,62 +10,133 @@ window.card = (function () {
     'bungalo': 'Бунгало',
   };
 
-  var renderCard = function (ad) {
-    var newCard = cardTemplate.cloneNode(true);
+  var renderCard = function (ad, pin) {
+    var newCard = cardTemplateElement.cloneNode(true);
     var popupCloseButtonElement = newCard.querySelector('.popup__close');
 
-    var cardCloseClickHandler = function (evt) {
-      evt.preventDefault();
-      newCard.remove();
-      document.removeEventListener('keydown', cardCloseEscPressHandler);
+    var cardTextElements = [
+      {
+        'element': newCard.querySelector('.popup__title'),
+        'text': ad.offer.title ?
+          ad.offer.title : '',
```

Comment:
Есть более красивая запись, если понимаешь её то можно пользоватся
```suggestion
        'text': ad.offer.title || '',
```